content/en/community/hackathons/usoc23/baby-steps/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Session 02 - Baby Steps
+linkTitle: Bulding Unikernels
+weight: 5
+summary: |
+  We start presenting the modular architecture of Unikraft and how students can build their unikernels using the manual, `make`-based approach.
+  Expected time: 100 minutes
+---
+
+{{< readfile file="/community/hackathons/sessions/setup/content/intro.md" markdown="true" >}}
+
+
+## Reminder
+In the previous [session](community/hackathons/usoc23/overview), we have experimented with building and running several unikernels using the scripts provided in this [repository](https://github.com/unikraft-upb/scripts).
```

Comment:
We don't use the script anymore, replace it with `instructions from the README.md files`.